packages/wp-dashboard/src/components/editorSettings/customFonts/index.js

```diff
@@ -242,6 +258,32 @@ function CustomFontsSettings({
     [handleOnSave]
   );
 
+  useEffect(() => {
+    const el = currentFontsRowsRef.current[`row-${currentFontsFocusIndex}`];
+    if (el) {
+      el.focus();
+      setCurrentFontsActiveId(el.id);
+    }
+  }, [currentFontsFocusIndex]);
+
+  const isListBoxActiveRow = (index) => currentFontsFocusIndex === index;
+
+  const handleListBoxNav = useCallback(
+    (evt) => {
+      const { key } = evt;
+      if (key === 'ArrowUp') {
+        evt.preventDefault();
+        setCurrentFontsFocusIndex((index) => Math.max(0, index - 1));
+      } else if (key === 'ArrowDown') {
+        evt.preventDefault();
+        setCurrentFontsFocusIndex((index) =>
+          Math.min(customFonts.length - 1, index + 1)
+        );
+      }
+    },
+    [customFonts]
+  );
```

Comment:
A comment might be useful here. This code isn't clear.

Reply:
Just pushed a comment --- let me know if it's helpful and or feel free to push a commit